docs/Example Schedules/CE Schedules/compe-kteal2.md

```diff
@@ -0,0 +1,185 @@
+# Computer Engineering, Statistics Minor, 7-8 Semesters
```

Comment:
Change title to "Robotics, Statistics Minor, 8 Semesters".  Add the Robotics tag at the top. (and whichever of the approved tags you think also apply)